tests/code_coverage/sample_fuzz_test/expected_coverage/fuzz+lib+codeCoverage-fuzz.json

```diff
@@ -104,7 +104,10 @@
 						"start": { "line": 24, "column": 1 },
 						"end": { "line": 26, "column": 2 }
 					},
-					{ "start": {}, "end": {} }
+					{
```

Comment:
I can see how this is the result of the workaround that was added but I don't know enough about coverage maps to see why this is the behavior we want. Or maybe flipped around, why was it being empty resulting in incorrect coverage info and why does this fix it?

Reply:
The linked ticket describes this in a little bit more detail. In the end istanbul expects a location with `start` and `end` coordinates for every branch, also implicit else branches, otherwise it does not process it's coverage.